### PR TITLE
Added NoSwing module.

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/main/java/net/ccbluex/liquidbounce/injection/mixins/minecraft/entity/MixinClientPlayerEntity.java [hunk 1]
```diff
@@ -189,4 +193,16 @@ private void injectForcedState(CallbackInfoReturnable<Boolean> cir) {
         }
     }
 
+    @Inject(method = "swingHand", at = @At("HEAD"), cancellable = true)
+    private void swingHand(Hand hand, CallbackInfo callbackInfo) {
+        if (ModuleNoSwing.INSTANCE.getEnabled()) {
+            if (ModuleNoSwing.INSTANCE.getEnabled()) {
```
Comment (1zun4secondary) [inline src/main/java/net/ccbluex/liquidbounce/injection/mixins/minecraft/entity/MixinClientPlayerEntity.java]: Checking if the module is enabled two times.

File: src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/render/ModuleNoSwing.kt [hunk 2]
```diff
@@ -0,0 +1,8 @@
+package net.ccbluex.liquidbounce.features.module.modules.render
```
Comment (1zun4secondary) [inline src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/render/ModuleNoSwing.kt:19]: Please always add the copyright header into files.

File: src/main/java/net/ccbluex/liquidbounce/injection/mixins/minecraft/entity/MixinClientPlayerEntity.java (hunk 1 above)
Comment (ghost) [reply src/main/java/net/ccbluex/liquidbounce/injection/mixins/minecraft/entity/MixinClientPlayerEntity.java]: holy moly did not see that

File: src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/render/ModuleNoSwing.kt (hunk 2 above)
Comment (ghost) [reply src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/render/ModuleNoSwing.kt:19]: sure, but why add copy right anyway?

**Pull request merged**